src/meta/src/hummock/manager/commit_epoch.rs

```diff
@@ -426,15 +426,17 @@ impl HummockManager {
         for (mut sst, group_table_ids) in sst_to_cg_vec {
             for (group_id, match_ids) in group_table_ids {
                 let group_members_table_ids = group_members_table_ids.get(&group_id).unwrap();
-                if match_ids
+                if sst
```

Comment:
It seems unnecessary and unclear to put this under the nested for loop. What we want to do here is to avoid SST split if all existing table ids of a sst belong to one and only one compaction group. Therefore, I suggest we put this check before L427 in the outer for loop. IIUC, we only need to check whether `group_table_ids.len() == 1`.

Reply:
Btw, do you think we need to modify `table_ids` and `sst_size` if `group_table_ids.len() == 1` but table_ids in SstableInfo is a super set of `group_member_table_ids` (i.e. L407 is hit)